### ISSUE TITLE
Response repeated cookies in headers

### ISSUE THREAD

**Issue by hhxsv5** [hhxsv5/laravel-s#107]
1. Tell us your software version
    | Software | Version |
    | --------- | --------- |
    | PHP | 5.6.4+ |
    | Swoole | 1.7.19+ |
    | Laravel/Lumen | 5.4+ |

2. Detail description about this issue(error/log)

    Response repeated cookies in headers
![image](https://user-images.githubusercontent.com/7278743/49143054-2ab32580-f335-11e8-95d9-7084825a7b24.png)

3. Give us a `reproducible` code block and `steps`

    ```PHP
    $response = response('test');
    $response->headers->setCookie(new \Symfony\Component\HttpFoundation\Cookie('test', time(), time() + 3600));
    return $response;
    ```

**Comment by hhxsv5** [comment]
Since the package  [http-foundation](https://github.com/symfony/http-foundation/blob/3.3/ResponseHeaderBag.php#L45) 3.2 to 3.3 is an incompatible upgrade, the headers contain SetCookie, but not in 3.2 and below.

**Comment by hhxsv5** [comment]
Fixed by [2fb5996](https://github.com/hhxsv5/laravel-s/commit/2fb5996b28fe6bc077f3d29fb81a7b2c54f81137), release in [v3.1.1](https://github.com/hhxsv5/laravel-s/releases/tag/v3.1.1).
